aws/client_test.go

```diff
@@ -0,0 +1,56 @@
+package aws
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/quay/claircore/test/integration"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Client_GetMirrors(t *testing.T) {
```

Comment:
The tests that handle the body should have an in-process http server spun up and at least be tested against a known-good body.